### PR TITLE
Minutes office hours Oct. 18th

### PR DESCRIPTION
**Pull request by lgblaumeiser**
## Description
Minutes of the office hour on Oct., 18th 2024

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [X] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [X] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

File: blog-meeting-minutes/2024-10-18-community-hour.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+- Deprecation of Repos. The regulation in TRG 7.09 is currently reworked to be more precise.
+- Discussion on a TRG about testing.
+  - Doubleslash prepares a TRG concerning automated testing
+  - Some repositories use SonarCloud and code coverage measurements there
+  - General openness towards a new trial on some specs concering testing
+
+### Community/Open Planning
+
+- Files like presentations done during Open Planning are currently not provided in Open Source. The proposal is to store such files in the already existing large files repository attached to the webpage repo.
+- Open Meetings page has been enhanced, there are now relevant links like to the board that is relevant, ...
```
Comment (stephanbcbauer) [inline blog-meeting-minutes/2024-10-18-community-hour.md:38]:
```suggestion
- Open Meetings page has been enhanced, there are now relevant links like to the task board that is relevant and related release timelines
- Regular New Joiner meeting was canceled. If a session is needed, it needs to be requested
```

**Review by stephanbcbauer — CHANGES_REQUESTED** [review]
LGTM just one suggestion related the canceled new joiner session

File: blog-meeting-minutes/2024-10-18-community-hour.md (hunk 1 above)
Comment (lgblaumeiser) [reply blog-meeting-minutes/2024-10-18-community-hour.md]: done

**Review by stephanbcbauer — APPROVED** [review]
LGTM ⇾ i don't miss any topics, maybe other @eclipse-tractusx/automotive-tractusx-committers ???

**Pull request merged**